catalog/installWeather.sh

```diff
@@ -23,6 +23,12 @@ install 'weather/forecast.js'       weather/forecast \
     -a sampleInput '{"latitude":"34.063", "longitude":"-84.217", "apiKey":"XXX"}' \
     -a sampleOutput '{"forecasts":[ {"dow":"Monday", "min_temp":30, "max_temp":38, "narrative":"Cloudy"} ]}'
 
+install 'weather/pollen.js'       weather/pollen \
+    -a description 'Weather.com pollen information' \
+    -a parameters '[{"name":"latitude", "required":true}, {"name":"longitude", "required":true}, {"name":"apiKey", "required":true, "type":"password", "bindTime":true}]'
```

Comment:
will do, but since I based it on the way forcast is installed, should I add zipcode there as well?